content/en/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md

```diff
@@ -11,17 +11,16 @@ weight: 10
 
 <!-- overview -->
 
-Network plugins in Kubernetes come in a few flavors:
+Network plugins in Kubernetes come only as CNI plugins:
 
 * CNI plugins: adhere to the [Container Network Interface](https://github.com/containernetworking/cni) (CNI) specification, designed for interoperability.
   * Kubernetes follows the [v0.4.0](https://github.com/containernetworking/cni/blob/spec-v0.4.0/SPEC.md) release of the CNI specification.
-* Kubenet plugin: implements basic `cbr0` using the `bridge` and `host-local` CNI plugins
 
 <!-- body -->
 
 ## Installation
 
-The kubelet has a single default network plugin, and a default network common to the entire cluster. It probes for plugins when it starts up, remembers what it finds, and executes the selected plugin at appropriate times in the pod lifecycle (this is only true for Docker, as CRI manages its own CNI plugins). There are two Kubelet command line parameters to keep in mind when using plugins:
+The kubelet has a single default network plugin, and a default network common to the entire cluster.The CRI manages its own CNI plugins. There are two Kubelet command line parameters to keep in mind when using plugins:
```

Comment:
This revised paragraph might need technical review. I don't think there is a default network plugin for the kubelet now that `dockershim` is gone.